scripts/build_distro

```diff
@@ -22,6 +22,17 @@ then
   PKG_CLEAN+=" ${CLEAN_EMU_32BIT}"
 fi
 
+# Device RK3566-X55 needs a built RK3466 root
+if [ "${DEVICE}" = "RK3566-X55" ]
```

Comment:
We don't want to hard code tests like this where it can be avoided.  It would be better to build the expected root path from variables and test if that exists.  If it doesn't, push the error message to the user using the variables and exit 1.  In this case the variable you want to inspect is "DEVICE_ROOT".  If the variable exists, but the path does not, exit 1.

Reply:
ah, thanks! I was unaware of the DEVICE_ROOT / DEVICE dependency.